src/packages/@app/pages/Collections/CollectionsPage.svelte

```diff
@@ -324,6 +329,16 @@
     onUploadFile={_viewModel.uploadFormFile}
     onExtractGitBranch={_viewModel.extractGitBranch}
   /> -->
+<ModalWrapperV1
+  title={"New Collection"}
+  type={"dark"}
+  width={"35%"}
+  zIndex={1000}
+  isOpen={isImportCollectionPopup}
+  handleModalState={(flag = false) => {
+    isImportCollectionPopup = flag;
+  }}
+>
```

Comment:
Why we have changed the structure of import collection, i think we have just put the modal wrapper outside, any specific reason for that?

Reply:
this is to handle re-renders that invokes animation every time modal opens